packages/components/psammead-episode-list/src/Image.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { string } from 'prop-types';
+import omit from 'ramda/src/omit';
+import styled from '@emotion/styled';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import { getMinion } from '@bbc/gel-foundations/typography';
+import { C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import {
+  GEL_SPACING_HLF,
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+import { withEpisodeContext } from './helpers';
+
+const Wrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  line-height: 0;
```

Comment:
what does this do?

Reply:
Deleted this 👍  Was originally to remove additional spacing below the image, but added the `line-height: 0`  to the list item instead